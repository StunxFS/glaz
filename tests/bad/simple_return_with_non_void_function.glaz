fn a() i32 { return; }

fn main() {}
