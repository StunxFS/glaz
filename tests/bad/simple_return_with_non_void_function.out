tests/bad/simple_return_with_non_void_function.glaz:1:15: error: simple `return` in a function whose return type is not void
     | 
   1 | fn a() i32 { return; }
     |              ^^^^^^
     = note: expected value of type `i32`
glaz error: could not compile `simple_return_with_non_void_function`, aborting due to previous error.
